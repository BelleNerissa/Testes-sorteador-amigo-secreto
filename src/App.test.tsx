import App from "./App";

const mockNavegacao = jest.fn()

jest.mock('react-router-dom', () => {
    return {
        useNavigate: () => mockNavegacao
    }
})

describe('o componente de cabeÃ§alho', () => {
    test('deve ser renderizada corretamente', () => {
        expect(<App />).toMatchSnapshot()
    })
})