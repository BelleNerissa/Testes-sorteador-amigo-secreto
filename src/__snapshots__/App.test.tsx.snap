// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`o componente de cabe√ßalho deve ser renderizada corretamente 1`] = `<App />`;
